deploy/docs/v2_migration_doc.md

```diff
@@ -82,3 +82,65 @@ the exact steps for migration.
 ## How to upgrade
 
 **Note: The below steps are using Helm 3. Helm 2 is not supported.**
+
+### 1. Upgrade to helm chart version `v1.3.2`
+
+Run the below command to fetch the latest helm chart:
+
+```bash
+helm repo update
+```
+
+For the users who are not already on `v1.3.2` of the helm chart, please upgrade
+to that version first by running the below command.
+
+```bash
+helm upgrade collection sumologic/sumologic --reuse-values --version=1.3.2
```

Comment:
could we test against `--reuse-values`? AFAIK changes in values.yaml won't be applied